src-docs/src/views/call_out/info.js

```diff
@@ -41,6 +41,14 @@ export default () => (
 
     <OuiSpacer size="m" />
 
+    <OuiCallOut
+      title="Callouts can be dismissed when dismissible is set to true unless the color is danger or warning. "
+      iconType="wrench"
+      dismissible="true"
```

Comment:
```suggestion
      dismissible
```
Since its a true value, this should be enough